### ISSUE TITLE
Land

### ISSUE THREAD

**Issue by nelsonic** [dwyl/phase-two#3]
![buy-land](https://cloud.githubusercontent.com/assets/194400/19157161/526e725e-8bdc-11e6-9fec-3b26f36d4489.png)

Checklist of things to Investigate (_before we can move forward with land purchase_)
+ [ ] Broadband (Fibre or Wireless) options.
+ [ ] Planning Permission. _specifically_ can we:
  + 1.re-build existing house ?
  + 2. add more container homes later ?
+ Water (will the council hook us up to the mains?? how much will it cost us?)
+ Goal / Business Plan (_how much detail is required_) ?

@iteles what _else_?

**Comment by nelsonic** [comment]
I feel a "_research_" trip coming soon ... 😉 
+ http://www.montesaraz.com/?ID=4
+ https://www.wonderful.land/montegois/

**Comment by iteles** [comment]
Adding to the top comment above so the list is all in one place, but more research:
http://www.casasbrancas.pt/rede/naturarte-rio/
http://www.casasbrancas.pt/rede/naturarte-campo/

**Comment by nelsonic** [comment]
yeah, [casasbrancas](http://www.casasbrancas.pt/) are pretty _sweet_... 
_however_, _we_ are going to take it to the `nextLevel();` 😉 
Which reminds me... we're going to need to leverage our funds... #5

**Comment by nelsonic** [comment]
I try not to "_spam_" you with _too_ many links to _potential_ properties, but this one is pretty good.
http://www.imovirtual.com/imoveis/quintas_e_herdades/quinta-ourique/5683604/
![land-ourique-with-lake](https://cloud.githubusercontent.com/assets/194400/20231377/2edb1286-a859-11e6-9b0c-5929d8674704.png)

Details:
![land-ourique-with-lake-detail](https://cloud.githubusercontent.com/assets/194400/20231387/57ac5cf6-a859-11e6-839e-f53a73aa0e8a.png)

Ourique https://en.wikipedia.org/wiki/Ourique is well-located for public transport (train/bus)
It's close to the `E1` highway and is aprox **1h** from ***Faro*** Airport and **2h** from **Lisbon**.

I've contacted the seller on the site but will make a phonecall tomorrow morning to see if we can view it tomorrow.

**Comment by nelsonic** [comment]
This land is _incredible_: http://www.imovirtual.com/imoveis/quintas_e_herdades/herdade-com-barragem/4194482/
It has a 7ha lake/dam which is fed by a river. the location is "_worse_" than other land we've seen but it's in the middle of a national park so the air is pretty fresh... Given that we are building "temporary structures" there won't be a "planning permission" issue ...

**Comment by c0rr314** [comment]
This is an incredible property. Such a huge lake AND a stream. Doesn't get much better than this!

**Comment by nelsonic** [comment]
Yeah, I was pretty _excited_ when I saw it, I just wanted to sell everything and go for it!
But I think a more _prudent_ approach is wise. We're going to spend the _bare minimum_ on a tiny piece of land and plant some trees and continue investing in our "Phase One" business for the foreseeable future... not that we won't have _amazing_ land, just not this _scale_... see: #25

**Comment by nelsonic** [comment]
A 10 Bedroom House with a Pool, BBQ Area, Enough Land to Graze Cattle (or start a solar farm!) is €350k ...
![image](https://user-images.githubusercontent.com/194400/27006161-64118fee-4e25-11e7-97b3-16764fef20ad.png)
![image](https://user-images.githubusercontent.com/194400/27006182-b743b7e6-4e25-11e7-9d1b-0ac0be8af4b6.png)
![image](https://user-images.githubusercontent.com/194400/27006192-cc528a4a-4e25-11e7-90ed-56f57f5b4179.png)
![image](https://user-images.githubusercontent.com/194400/27006183-bc41573a-4e25-11e7-8f23-c2a48652fdf4.png)


The price of a Studio Apartment in an OK area of London ...
![image](https://user-images.githubusercontent.com/194400/27006178-a39c2df4-4e25-11e7-87cf-890e89c6e9d6.png)

🤣

**Comment by carolrmc** [comment]
no brainer!

**Comment by nelsonic** [comment]
https://www.imovirtual.com/comprar/quintaeherdade/?order=filter_float_m%3Adesc&nrAdsPerPage=72

**Comment by nelsonic** [comment]
This is the link to view the latest _large_ parcels of Land in PT:
https://www.imovirtual.com/comprar/quintaeherdade/?nrAdsPerPage=72&search%5Border%5D=filter_float_m%3Adesc

**Comment by nelsonic** [comment]
[![image](https://user-images.githubusercontent.com/194400/40231428-0d7e04e8-5a93-11e8-96b2-da723096e8a4.png)](https://youtu.be/ZSPkcpGmflE)
https://youtu.be/ZSPkcpGmflE

**Comment by nelsonic** [comment]
https://www.pureportugal.co.uk/property/shb999-the-mill/
![4FD18219-0E87-4531-BACD-342488CD18D5](https://user-images.githubusercontent.com/194400/156842551-298d4542-3ce1-4fe9-8b0d-3a457d078e87.png)

![F18DE11E-A117-4C80-B564-70E828E17DA6](https://user-images.githubusercontent.com/194400/156842569-411ce668-a9aa-44e2-8337-a9aab8961d8b.png)

**Comment by iteles** [comment]
This sounds absolutely amazing 😱

**Comment by nelsonic** [comment]
As you can imagine, I’m obvs keen. hence sharing it. 
It’s not a *huge* amount of land. But 3ha is roughly 3 football pitches:

https://www.thecalculatorsite.com/articles/units/how-big-is-a-hectare.php

![B6401C6A-CCA8-4287-A920-4DE2F292FBB1](https://user-images.githubusercontent.com/194400/156882895-f8b32f50-edf4-4b1a-9f9c-969ad80a14a7.jpeg)

There is a local primary car health center 5 mins away: https://pt.m.wikipedia.org/wiki/Centro_de_Sa%C3%BAde_de_Monchique

It’s 25mins drive from the nearest public (full service) hospital in Portimão.

**Comment by iteles** [comment]
Let me know if you want to take a couple of nights to go and see it! (though 4 night travels in the car within one week will definitely be pretty disruptive for Alex 😭) 

Pretty exciting!!

![image](https://user-images.githubusercontent.com/4185328/157984443-9aba052c-3bfc-41e4-8d35-5208b9cdd3d6.png)

**Comment by nelsonic** [comment]
WoW! You have a nice car. 😜   (_this isn't `public` anywhere BTW ..._)
But yes, I'd love to go visit the land. (_might need to take a couple of days off if the internet is too weak to work_)

**Comment by nelsonic** [comment]
@iteles still keen to visit the land. Did you ever hear back from them? ⏳

**Comment by nelsonic** [comment]
@iteles wanna try and view it next time we go down south? Or want to look for something closer to Braga/Landim?

**Comment by nelsonic** [comment]
@iteles want to go look at this land when we head down to LX for "The Last Time To See me Before I Die!" ...?

BTW: the gig is available on YT: https://github.com/nelsonic/nelsonic.github.io/issues/860

**Comment by iteles** [comment]
Para fazer isso temos de ir 'com tempo'. We're putting Alex through two nights where his most restorative sleep period (first sleep off the night) is taking place in a car seat just to get to Lisbon and back. 

To get to Monchique it's a 3+ hour drive so we either leave Alex with grandparents and drive down and up by ourselves on the same day or we disrupt his sleep for 2 _more_ nights by driving at the end of the day. 

The short answer is yes, absolutely, but I'm not sure if we're not better off going alone so as not to destroy Alex's schedule just before the wedding we're attending.

**Comment by nelsonic** [comment]
Ok. let's leave it for now then. Don't want to disrupt 👶 ... and its also probably best to _focus_ on finishing [`@home`](https://github.com/dwyl/home/issues/14) first. 🏡  👍 
My only _reasoning_ for even _considering_ buying land before the house is _finished_ is to plant trees ... #6 / #28 
But I think we need to sit down and draft a better _plan_ for the trees up here.

**Comment by nelsonic** [comment]
RIVERSIDE LAND WITH 2HA, 2 HOUSES, RUIN, OUTBUILDING & WATERMILL FOR RENOVATION
https://www.pureportugal.co.uk/property/imo270-watermill-farm/
![image](https://user-images.githubusercontent.com/194400/187411640-02f8884e-19f6-4f5a-a4f8-bad4cbc619ec.png)

![image](https://user-images.githubusercontent.com/194400/187411698-3d119ba4-36db-4721-b4dc-246642384243.png)

Castelo de Paiva: https://goo.gl/maps/Dg5zh27QVieqzpg96
![image](https://user-images.githubusercontent.com/194400/187412050-82a037b0-be72-4dbc-860e-aef7fb8b0376.png)

**Comment by iteles** [comment]
DEFINITELY worth seeing.

**Comment by nelsonic** [comment]
Viewing booked for tomorrow Wednesday 31<sup>st</sup> August **14:00**: https://github.com/dwyl/phase-two/issues/25#issuecomment-1230525562  📆

**Comment by nelsonic** [comment]
"_things to consider and costly mistakes to avoid when looking for land to start your journey_": https://youtu.be/v02JLhjN2Vw
<img width="755" alt="image" src="https://user-images.githubusercontent.com/194400/188796541-53a052ff-0c71-4870-956f-2bd1fe182143.png">

His channel might be called "**The _Dutch_ Farmer**" [**`283K subscribers`**!] but they live in ***Portugal***.
Highly recommend watching this video and seeing how #116 ticks _all_ the boxes.
Most of the info is covered in the issues we've opened in this repo. 

This was the most up-voted comment on the video:

<img width="711" alt="image" src="https://user-images.githubusercontent.com/194400/188800049-a8a2fbe9-70c5-4d14-9cc3-0f9f61be4430.png">


The **one** thing we need to if we move forward with #116 is request the "**pedido de informação prévia**" (request for prior information) from the council to know as much as possible about the land beforehand. e.g. any previous planning permission requests that have been denied. Any outstanding debts and get it in writing what is allowed (or not) on the land.
